Framework/Test_Framework_UnitTest.rb

```diff
@@ -0,0 +1,240 @@
+require 'test/unit'
+require_relative 'Selenium_FrameWork'
+
+class Test_TestFrameWork < Test::Unit::TestCase
+
+  def test_make_framework_object
+    create_testframe_instance = SeleniumFrameWork.new()
+    assert(create_testframe_instance.instance_of? SeleniumFrameWork)
+    create_testframe_instance.close_site()
+  end
+
+  def test_navigate_url
+    create_testframe_instance = SeleniumFrameWork.new()
+    url = "file://src/Test_Framework.html"
+    assert_equal("Site Navigated", create_testframe_instance.open_site(url))
+    create_testframe_instance.close_site()
+  end
+
+  def test_navigate_invalid_url
+    create_testframe_instance = SeleniumFrameWork.new()
+    url = "file:///src/abc"
+    # url = "http://127.0.0.1:5500/Framework/abc.html"
+    assert_equal("Site Not Found", create_testframe_instance.open_site(url))
+    create_testframe_instance.close_site()
+  end
+
+  def test_close_driver
+    create_testframe_instance = SeleniumFrameWork.new()
+    assert_equal("Diver Closed", create_testframe_instance.close_site())
```

Comment:
A better way to check if the driver does not exist anymore is by using `driver.nil?`